sklearn/ensemble/voting_classifier.py

```diff
@@ -252,17 +269,29 @@ def transform(self, X):
         else:
             return self._predict(X)
 
+    def set_params(self, **params):
+        """ Setting the parameters for the voting classifier
+
+        Valid parameter keys can be listed with get_params().
```

Comment:
Can you add under the `get_params` heading? I would just say "Get the parameters of the VotingClassifier". In addition, I would also document the parameter deep saying that setting it to True gets the various classifiers and the parameters of the classifiers as well.